packages/artillery-plugin-publish-metrics/lib/open-telemetry/metrics.js

```diff
@@ -45,6 +44,7 @@ class OTelMetricsReporter {
   }
 
   configure(config) {
+    this.debug = require('debug')(`plugin:publish-metrics:${this.config.type}`);
```

Comment:
Currently OTel reporter `metrics` doesn't support any vendor-specific reporters, but for future proofing we require dynamically.